packages/modules/manage-versions/src/components/ManageVersions/VersionsTab/VersionsTab.tsx

```diff
@@ -66,6 +67,49 @@ const VersionsTab = (props: VersionsTabProps) => {
     }
   }
 
+  const [openMenuId, setOpenMenuId] = useState(null);
```

Comment:
it didnt work without managing the state?

Reply:
@veekeys It is working, I have removed controlled Dropdown logic now